treeherder/etl/management/commands/pulse_listener.py

```diff
@@ -0,0 +1,37 @@
+import environ
+from django.core.management.base import BaseCommand
+from treeherder.services.pulse import (PulsesConsumer,
+                                       PushConsumer,
+                                       TaskConsumer,
```

Comment:
What's the reason to import these two consumers?

Reply:
Will remove them. Was just first checking with my new consumer!